connectors/src/connectors/slack/chat/blocks.ts

```diff
@@ -212,7 +212,7 @@ export function makeErrorBlock(
         type: "section",
         text: {
           type: "plain_text",
-          text: errorMessage,
+          text: truncate(errorMessage, MAX_SLACK_MESSAGE_LENGTH),
```

Comment:
The error message was too long which leads to a `msg_too_long` error ([logs](https://app.datadoghq.eu/logs?query=%22Unexpected%20exception%20answering%20to%20Slack%20Chat%20Bot%20message%22%20OR%20%22Failed%20answering%20to%20Slack%20Chat%20Bot%20message%22&agg_m=count&agg_m_source=base&agg_t=count&cols=host%2Cservice%2C%40time&event=AwAAAZL9YjjX_Q4WXQAAABhBWkw5WWtlNUFBRHJ2VExYdVo3LTNRQkYAAAAkMDE5MmZkNzgtMTNhYi00ZGZlLTgxMGItNTMwNjk1YTNkZWYzAAAd2A&fromUser=true&messageDisplay=inline&refresh_mode=paused&storage=hot&stream_sort=desc&viz=stream&from_ts=1730827670058&to_ts=1730828275666&live=false)).